app/src/main/java/com/example/makeitso/screens/login/LoginViewModel.kt

```diff
@@ -17,25 +17,23 @@ limitations under the License.
 package com.example.makeitso.screens.login
 
 import androidx.compose.runtime.mutableStateOf
-import androidx.lifecycle.viewModelScope
 import com.example.makeitso.LOGIN_SCREEN
 import com.example.makeitso.SETTINGS_SCREEN
-import com.example.makeitso.R.string as AppText
 import com.example.makeitso.common.ext.isValidEmail
 import com.example.makeitso.common.snackbar.SnackbarManager
 import com.example.makeitso.model.service.AccountService
-import com.example.makeitso.model.service.LogService
 import com.example.makeitso.model.service.StorageService
+import com.example.makeitso.model.service.LogService
 import com.example.makeitso.screens.MakeItSoViewModel
 import dagger.hilt.android.lifecycle.HiltViewModel
-import kotlinx.coroutines.launch
 import javax.inject.Inject
+import com.example.makeitso.R.string as AppText
 
 @HiltViewModel
 class LoginViewModel @Inject constructor(
     private val accountService: AccountService,
     private val storageService: StorageService,
```

Comment:
The `StorageService` is no longer used, so we can remove it from the `constructor`.